recipes/go-task/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set goname = "github.com/go-task/task" %}
+{% set name = "go-task" %}
+{% set version = "3.24.0" %}
+
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
```

Comment:
What's the reason to not use purely `src`?

Reply:
I am no go developer but my understanding is that go has strict directory requirements and expects a full namespace like `src/github.com/go-task`.
I took inspiration from [the kubo feedstock](https://github.com/conda-forge/kubo-feedstock/blob/d070776c6e917bf1c699a39f5aeb2f7457c37f34/recipe/meta.yaml#L11) as suggested by @dbast.